### DISCUSSION TITLE
Screen flicker - Android Pixel 8 Pro - Amcrest IP8M-DLB2998EW-AI (4096X1856)

### DISCUSSION THREAD

**Discussion by bwze** [blakeblackshear/frigate#14695] [Ask A Question] [answered]
I recently purchased and integrated an Amcrest Amcrest IP8M-DLB2998EW-AI into Frigate with very few hiccups along the way. The camera is a dual lens model with a 4K resolution....so 4096X1856.

Everything is working as I would expect it except for a notorious flickering issue that is present when viewing either the live feed for this camera or it's history with a Pixel 8 Pro using Chrome. 

When interacting with the feed, I've noticed that if I use my finger to either zoom or move the camera feed off the side of the screen a bit, the flickering stops but as soon as I unzoom or release the feed, it immediately starts to flicker again. 

Things I've tested:
 - I've tested this on an iPhone 14 and I see no flicker there so it seems to be related to the Pixel only. 
 - I've downloaded the Firefox browser on the Pixel and it does not exhibit the flickering issue.

Curious if anyone else has experienced this or if there is something easy that I'm missing as a fix?

**Comment by bwze** [comment]
Bumping once... I've tried to grab a screen recording but the flickering doesn't show up. I did record the issue with another phone but I'm reluctant to post it due to the quality. I was hoping the problem was something simple that I was overlooking.

**Reply by NickM-27** [reply]
I have a hikvision dual lens camera and I have not seen this

**Reply by bwze** [reply]
Would you mind sharing your config for that camera?

**Comment by bwze** [comment, marked as answer]
Found the problem. It was related to a phone setting on the Pixel 8 Pro. In Settings - Display, there is an option for Screen Resolution. My phone was defaulted to High Resolution (1008 x 2244). I changed it to Full Resolution (1344 x2992) and that fixed it.
